channeldb/payment_control_test.go

```diff
@@ -682,7 +690,9 @@ func TestPaymentControlMultiShard(t *testing.T) {
 		}
 
 		assertPaymentIndex(t, pControl, info.PaymentIdentifier)
-		assertPaymentStatus(t, pControl, info.PaymentIdentifier, StatusInFlight)
+		assertPaymentStatus(
+			t, pControl, info.PaymentIdentifier, StatusInitiated,
```

Comment:
I noticed that in the previous commit, `TestPaymentControlMultiShard` fails. So you probably want to move some of these fixes to earlier commits to keep them passing unit tests.

Reply:
ok moved the test fixes.